video/out/vo.c

```diff
@@ -132,7 +132,7 @@ struct vo_internal {
     bool hasframe_rendered;
     bool request_redraw;            // redraw request from player to VO
     bool want_redraw;               // redraw request from VO to player
-    bool send_reset;                // send VOCTRL_RESET
+    uint8_t send_reset;             // send VOCTRL_RESET (1) or VOCTRL_FRAME_RESET (2)
```

Comment:
Personally, I think this is more easily understood if it's kept separate from `send_reset` (`frame_reset` or something).

Reply:
bitmask have some ergonomic advantages here, but what I'm doing is half-assed.
can change if you want.